src/request_handler_proxy.cpp

```diff
@@ -0,0 +1,130 @@
+//
+// request_handler_proxy.cpp
+// ~~~~~~~~~~~~~~~~~~~
+//
+
+#include "request_handler_proxy.hpp"
+#include <string>
+#include <boost/lexical_cast.hpp>
+#include "http_client.hpp"
+#include "response.hpp"
+#include "request.hpp"
+#include "request_handler.hpp"
+
+namespace http {
+namespace server {
+
+RequestHandler::Status ProxyHandler::Init(const std::string& uri_prefix, const NginxConfig& config) {
+	uri_prefix_ = uri_prefix;
```

Comment:
These lines look to be using tabs for spacing, while the ones below them use spaces (very nit picky hahaha)